src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    operate(callback, number) {
+      callback(number);
+
+      return this;
+    },
+
+    add(number) {
+      calculator.result += number;
+    },
```

Comment:
Make it like a pure function.
Let this method return you a pure result, and save it in your `operate` method.
For example
```
add(num1, num2) {
	return num1 + num2;
}

Reply:
got it👍